backend/.eslintrc.js

```diff
@@ -1,17 +1,27 @@
 module.exports = {
   parser: "@typescript-eslint/parser",
   extends: [
+    "plugin:react/recommended",
     "plugin:@typescript-eslint/recommended",
     "prettier/@typescript-eslint",
     "plugin:prettier/recommended",
   ],
   parserOptions: {
     ecmaVersion: 2018,
     sourceType: "module",
+    ecmaFeatures: {
+      jsx: true,
+    },
   },
   rules: {
     "no-var": "error",
     "@typescript-eslint/no-explicit-any": "off",
-    "@typescript-eslint/no-unused-vars": "off"
+    "@typescript-eslint/no-unused-vars": "off",
+    "@typescript-eslint/explicit-function-return-type": "off"
```

Comment:
Why turn this off? I personally would prefer to have this feature on (though I'm not sure it is by default)

Reply:
Oh I see it was probably included in react/recommended. If this caused errors then probably ok to just leave it off for now and we can turn it on later if we want to